server/app/utils/analysis.py

```
@@ -0,0 +1,28 @@
+
+class SpermAnalyzerService:
+    def __init__(self, model_path: str | None = None) -> None:
+        self.model_path = model_path
+        self.yolo = None
+        try:
+            from ultralytics import YOLO  # type: ignore
+            if model_path:
+                self.yolo = YOLO(model_path)
+            else:
+                self.yolo = YOLO("yolov8n.pt")
+        except Exception:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)